gnd/src/main/res/values/strings.xml

```diff
@@ -16,128 +16,129 @@
 
 <resources>
 
-  <!-- Application name. Do not translate. -->
-  <string name="app_name" translatable="false">Ground</string>
-
-  <!-- Message shown while project is loading. -->
-  <string name="project_loading_please_wait">Project loading, please wait &#8230;</string>
-
-  <!--
-    ADD/EDIT FEATURE DATA
-  -->
-
-  <!-- Title shown above list of feature types when (+) is clicked on map. -->
-  <string name="add_feature_select_type_dialog_title">What would you like to add?</string>
-  <!-- Text shown to dismiss "Add a feature" dialog -->
-  <string name="add_feature_cancel">Cancel</string>
-
-  <!-- Shown in toolbar when adding a new feature and form instance. -->
-  <string name="add_feature_toolbar_title">Add a feature</string>
-  <!-- Shown in toolbar when editing an existing record (form instance). -->
-  <string name="edit_feature_data_toolbar_title">Edit feature data</string>
-  <!-- Shown in toolbar when adding a new record (form instance) to a feature. -->
-  <string name="add_feature_data_toolbar_title">Add feature data</string>
-  <!-- Shown in toolbar to save changes to feature data. -->
-  <string name="save">Save</string>
-  <!-- Shown when leaving edit mode without saving changes. -->
-  <string name="unsaved_changes">Save changes before closing?</string>
-  <string name="save_unsaved_changes">Save</string>
-  <string name="continue_editing">Continue editing</string>
-  <string name="close_without_saving">Close without saving</string>
-  <!-- Shown on save when connection was not available. -->
-  <string name="save_queued">Saved locally, queued for sync.</string>
-  <!-- Shown on save when connection is available (TBD). -->
-  <string name="saved">Changes saved</string>
-
-  <!--
-    FEATURE DATA FORMS
-  -->
-
-  <!-- Text in multiple-choice popup dialogs that when clicked, applied new selection to form -->
-  <string name="apply_multiple_choice_changes">Apply</string>
-  <!-- Text in multiple-choice popup dialogs that when clicked, discards any changes made. -->
-  <string name="discard_multiple_choice_changes">Cancel</string>
-  <!-- Shown below fields that are required but left empty (TBD). -->
-  <string name="required_field">This field is required</string>
-
-  <!--
-    VIEW FEATURE DATA
-  -->
-
-  <!-- Shown in record overflow menu to open popup dialog to edit feature metadata. -->
-  <string name="edit_record">Edit record</string>
-  <!-- Shown in record overflow menu to delete a record. -->
-  <string name="delete_record">Delete record</string>
-  <!-- Confirmation shown when deleting a record (TBD). -->
-  <string name="delete_record_confirmation">
+    <!-- Application name. Do not translate. -->
```

Comment:
Any idea why whitespace changed here? Was it committed w/the wrong white space? (we really need to add auto lint checks to PRs!)

Reply:
Ah, I ran format on the file after modifying it--that's the reason it changed, I guess it had "incorrect" formatting at some point.